fgd/brush/func/func_slick.fgd

```diff
@@ -0,0 +1,6 @@
+@SolidClass 
+	appliesto(+MOMENTUM, -engine) 
+= func_slick: "An entity that lets the player slide on selected brushes." + " Use func_slide for customization." 
```

Comment:
This entity is for the defrag slick mechanic, it's not interchangeable with func_slide. Otherwise I think it looks good.